system/security/zcrypt/zcrypt.cpp

```diff
@@ -1089,7 +1109,7 @@ char* gzip( const char* inputBuffer, unsigned int inputSize, unsigned int* outle
     // Create output memory buffer for compressed data. The zlib documentation states that
     // destination buffer size must be at least 0.1% larger than avail_in plus 12 bytes.
     const unsigned long outsize = (unsigned long)floorf((float)inputSize * 1.01f) + 12;
```

Comment:
unrelated: this is calculating 1% extra rather than 0.1% extra - and would probably be better avoiding floating point.